third-party/hwloc/Makefile

```diff
@@ -70,8 +70,9 @@ hwloc-config: FORCE
 # newer than $(HWLOC_SUBDIR)/doc/doxygen-doc/hwloc.tag prevents make from
 # trying to do that step.
 #
-	touch -m -r  $(HWLOC_SUBDIR)/doc/doxygen-doc/hwloc.tag -d '+1 sec' $(HWLOC_SUBDIR)/README 2>/dev/null || \
-		touch -m -r $(HWLOC_SUBDIR)/doc/doxygen-doc/hwloc.tag -A '01' $(HWLOC_SUBDIR)/README
+	-touch -m -r  $(HWLOC_SUBDIR)/doc/doxygen-doc/hwloc.tag -d '+1 sec' $(HWLOC_SUBDIR)/README 2>/dev/null || \
+		touch -m -r $(HWLOC_SUBDIR)/doc/doxygen-doc/hwloc.tag -A '01' $(HWLOC_SUBDIR)/README 2>/dev/null || \
+		touch $(HWLOC_SUBDIR)/README
```

Comment:
I probably got the polarity wrong on the call, but thought you'd said you were planning on using `touch -c` here?

Reply:
I was going to originally, but opted not to for more simplicity (this file exists, so there's no reason to worry about not creating it if it doesn't)

Reply:
Yeah, I heard that also. But I believe it's actually better not to have the `-c` here, because if the file doesn't exist and we also don't create it, then `make` will run `doxygen` to create it, and that's exactly what we're trying to prevent. So I'm going to approve as is.